static/src/javascripts/projects/common/modules/lazyload.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+import fetchJSON from 'lib/fetch-json';
+import fastdom from 'lib/fastdom-promise';
+import { identity } from 'lib/identity';
+
+const lazyload = (
+    url: string,
+    container: Element,
+    _options?: Object
```

Comment:
Would be nice to describe the possible propertries of the object at the same time.